test/images/scripts/lib/make-srpm

```diff
@@ -11,21 +11,13 @@ if [ -z "$version" ]; then
 fi
 
 modify_spec() {
-case $version in
-wip)
-sed '1i\
-%define gitcommit wip'
-;;
-*)
-sed "1i\
-%define tag $version"
-esac
+sed -e "1i\
+%define gitcommit wip\n%define tag $version\n"
```

Comment:
> Do all the images need to be regenerated when files here change?

No, I don't think so.  We use the sources to pre-install build dependencies only.